tools/toolchain/scripts/install_libvdwxc.sh

```diff
@@ -67,15 +68,15 @@ case "$with_libvdwxc" in
                 ./configure \
                     --prefix="${pkg_install_dir}" \
                     --libdir="${pkg_install_dir}/lib" \
-                    --with-fftw3 \
+                    --with-fftw3=${FFTW_ROOT} \
```

Comment:
have you seen a specific error triggering this change? I'm asking because we see a strange build failure on CentOS 7 with a custom GCC in libvdwxc due to some missing symbol in `fftw_mpi`.
And in principle this is not needed since we set `CPATH` and `LIBRARY_PATH` in the environment (at least when building fftw ourselves, didn't check what happens if someone provides a system fftw).

Reply:
All these fixes were needed for a successful toolchain built on our Merlin6 cluster (CPU with AVX512 capabilities and selinux fully enforced)

Reply:
ah, so exactly the case with the system-provided fftw, thanks!